applications/ContactStructuralMechanicsApplication/tests/ALM_frictionless_contact_test_2D/hertz_simple_test_parameters.json

```diff
@@ -83,13 +95,13 @@
             "max_number_results"          : 1000,
             "type_search"                 : "InRadius",
             "integration_order"           : 2,
-            "active_check_factor"         : 1.0e-02,
+            "active_check_factor"         : 1.0e-0,
```

Comment:
Does it work with the defaults?
If so it is best if you don't overwrite them, so that if you change them the change will be tested

Reply:
Ok